02_configure_host.sh

```diff
@@ -11,12 +11,13 @@ source ocp_install_env.sh
 # Note we copy the playbook so the roles/modules from tripleo-quickstart
 # are found without a special ansible.cfg
 export ANSIBLE_LIBRARY=./library
+export VM_NODES_FILE=${VM_NODES_FILE=:-'tripleo-quickstart-config/metalkube-nodes.yml'}
```

Comment:
Typo here, message from CI
Apr 12 11:02:50 |  [0;31mCould not find or access '/home/notstack/dev-scripts/:-tripleo-quickstart-config/metalkube-nodes.yml' on the Ansible Controller.[0m

Reply:
Oops, thanks!